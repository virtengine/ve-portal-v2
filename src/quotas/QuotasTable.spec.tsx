import { render } from 'enzyme';

import { QuotasTable } from './QuotasTable';

const quotas = require('./QuotasTable.fixture.json');

const renderComponent = (resource) => {
  const wrapper = render(<QuotasTable resource={resource} />);
  return wrapper.html();
};

describe('QuotasTable', () => {
  it('renders table for quotas', () => {
    const html = renderComponent({ quotas });
    expect(html).toMatchSnapshot();
  });
});
