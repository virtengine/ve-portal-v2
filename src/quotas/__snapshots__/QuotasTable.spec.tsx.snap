// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuotasTable renders table for quotas 1`] = `"<div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Floating IP count:</div><div class=\\"details-table__value\\">6 of 50</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Instances count:</div><div class=\\"details-table__value\\">5 of 30</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Network count:</div><div class=\\"details-table__value\\">1 of 100</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">RAM:</div><div class=\\"details-table__value\\">5 GB of 128 GB</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Security group count:</div><div class=\\"details-table__value\\">8 of 100</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Security group rule count:</div><div class=\\"details-table__value\\">14 of 100</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Snapshots count:</div><div class=\\"details-table__value\\">2 of 50</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Snapshots size:</div><div class=\\"details-table__value\\">50 GB of &#x221E;</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Storage:</div><div class=\\"details-table__value\\">295 GB of 295 GB</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Subnet count:</div><div class=\\"details-table__value\\">1 of 100</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">vCPU count:</div><div class=\\"details-table__value\\">3 of 64</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Volumes count:</div><div class=\\"details-table__value\\">8 of 50</div></div><div class=\\"details-table__row\\"><div class=\\"details-table__key\\">Volumes size:</div><div class=\\"details-table__value\\">245 GB of &#x221E;</div></div>"`;
